packages/core/admin/_internal/node/createBuildContext.ts

```diff
@@ -74,6 +74,16 @@ interface BuildContext {
    */
   target: string[];
   tsconfig?: CLIContext['tsconfig'];
+  /**
+   * Features object with future flags
+   */
+  features?: {
+    [key: string]: unknown;
+    future?: {
+      unstable_contentReleases: boolean;
+      [key: string]: unknown;
+    };
+  }
```

Comment:
```suggestion
  /**
   * Features object with future flags
   */
  features?: RenderAdminArgs['features']
```

You can import the type (make sure you `import type`) similar to how we import the DefaultDocument in the `staticFiles` file. Also, nitpick, but please put it in alphabetical order lol